src/core/storage/secureStorage/secureStorage.ts

```diff
@@ -6,8 +6,11 @@ import {
 enum KeyStoreKeys {
   APP_PASSCODE = "app-login-passcode",
   APP_OP_PASSWORD = "app-operations-password",
+  PASSWORD_SKIPPED = "app-password-skip",
```

Comment:
I already commented this on the last PR that got reverted. The secure storage is for secrets. This is not a secret.

@bao-sotatek can you support Duke here on how to use the basic/misc storage for these items

Reply:
I move `PASSWORD_SKIPPED` from secure storage to basic/misc storage.
Please help me check it.